src/static/js/ethers_helper.js

```diff
@@ -1205,7 +1207,7 @@ async function getCurveToken(app, curve, address, stakingAddress, minterAddress)
       unstaked: unstaked  / 10 ** decimals,
       contract: curve,
       tokens : [address, coin0],
-      token,
+      token0,
```

Comment:
This might (or might not) break the price calculations so needs to be tested in some curve projects